projects/cds/src/merchandising/model/merchandising-user-context.model.ts

```diff
@@ -1,5 +1,5 @@
 export interface MerchandisingUserContext {
   category?: string;
-  productId?: string;
+  products?: string[];
```

Comment:
same as above

Reply:
We may be able to update this here, but then we wouldn't be able to use the spread operator when building up the `StrategyRequest` so it would complicate the code slightly. Thoughts?

Reply:
If it is to avoid complications, it is fine, just put a comment that it is the productCodes of OCC (along those lines)